which_language.md

```diff
@@ -190,3 +190,46 @@ Materials
 - [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS)
 - [JS for C & Python programmers](https://www.wooji-juice.com/blog/javascript-article.html)
 
+Lua
+==========
+A light, somewhat verbose Python-like language where indentation doesn't matter and almost everything is a table. Also, indexing starts from one. Typically, the Lua VM sits on top of a lower-level engine, providing an easy-to-use scripting interface.
```

Comment:
Python-like could be misleading here.  I see what you mean but if the target audience is people who do not know what "a table" is, this is overly detailed.  Same with "indexing starts from one", as I would hope reading the intro doesn't require too much (any) technical knowledge.

Also it shouldn't be required to know what a "VM" is to read the intro